syft/serde.py

```diff
@@ -474,7 +474,7 @@ def _detail_slice(my_slice: Tuple[int, int, int]) -> slice:
     This function extracts the start, stop and step from a list.
 
     Args:
-        list: a list defining the slice parameters [start, stop, step]
+        my_slice (tuple): a list defining the slice parameters [start, stop, step]
```

Comment:
A tuple defining the slice parameters?

Reply:
this one is correct

Reply:
This was a comment on the text of the comment